### ISSUE TITLE
TST: Passing a frame as a positional argument is now deprecated

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#7829]
Excerpt from https://travis-ci.org/bsipocz/astropy/jobs/429122972 . Looks like code was updated but not the tests, @eteq and @adrn ?
```
astropy/coordinates/tests/test_sky_coord.py::
    test_round_tripping[ICRS-ICRS-None-None-J1980.0-J1980.0]
  .../coordinates/sky_coordinate.py:1786: AstropyDeprecationWarning:
    Passing a frame as a positional argument is now deprecated,
    use the frame= keyword argument instead.
  "instead.", AstropyDeprecationWarning)
```

**Comment by abhinuvpitale** [comment]
So the change needed in the tests is to replace 
[this](https://github.com/astropy/astropy/blob/master/astropy/coordinates/tests/test_sky_coord.py#L96)
with 
`sc = SkyCoord(frame=frame0, RA, DEC, **attrs0)`

**Comment by eteq** [comment]
Yes, that is correct @pllim.  Appears to be 4 years old, though, so good catch!

@abhinuvpitale - you'd actually need to do ``sc = SkyCoord(frame=frame0, ra=RA, dec=DEC, **attrs0)`` or ``sc = SkyCoord(RA, DEC, frame=frame0, **attrs0)``, because a positional argument cannot come after a keyword argument.

**Comment by abhinuvpitale** [comment]
Okay, taking it up and adding the keyword arguments !

**Comment by pllim** [comment]
Fixed by #7869. Thanks, @abhinuvpitale !
